### PR TITLE
feat: progress in keycloak admin

### LINKED ISSUE #280: Pull account information from Keycloak programatically
Keycloak <-> DAISY integration - pulling OIDC's IDs of users

### PR DESCRIPTION
**Pull request by jLebioda**
closes #280

### CODE REVIEW

**Comment by jLebioda** [comment]
@vildead I believe that we could already merge this bad boy, and address any other improvements in dedicated issues

File: core/utils.py [hunk 1]
```diff
@@ -22,3 +24,25 @@ def wrap(*args, **kwargs):
...
+    """
+    def __init__(self) -> None:
+        pass
+
+    def _check_idservice(self):
+        idservice_endpoint = getattr(settings, 'IDSERVICE_ENDPOINT', None)
+        # TODO: check ID service endpoint connectivity
+        # i.e.: make http GET request 
+
+        # Checks if the function can be importable
```
Comment (vildead) [inline core/utils.py:42]: small typo - can be fixed once the method is finished

**Review by vildead — APPROVED** [review]
Nice! First time I see abstract classes in real. And the doc of variables in settings.py is also nice addition.
I have two points we may keep for further discussions:
1. as opensource project, naming packages/modules `lcsb` does not seem right. 
2. contact and user could inherit (be) same class  - contact would just not have login right

File: core/utils.py (hunk 1 above)
Comment (jLebioda) [reply core/utils.py:42]: Ahh, adding this file was a mistake, it is by no means ready

**Comment by jLebioda** [comment]
Thanks!

> 1. as opensource project, naming packages/modules `lcsb` does not seem right.
You are right! Ideally they should be in a separate django application in a separate directory. For now I've tried to keep them in a separate file so that they are not interleaved.

> 2. contact and user could inherit (be) same class  - contact would just not have login right
This is a bit confusing part - everything would be much easier if it was one class or if we could use abstractions (but databases don't like that idea)

**Pull request merged**